nixos/tests/openssh.nix

```diff
@@ -152,6 +152,36 @@ in {
         ];
       };
 
+    server-sftp =
+      { pkgs, ... }:
+      {
+        environment.etc."ssh/authorized_keys/alice" = {
+          mode = "0644";
+          text = snakeOilPublicKey;
+        };
```

Comment:
Is there a reason why you don't use `users.extraUsers.alice.openssh.authorizedKeys`?

Reply:
I assumed this was necessary with the `ChrootDirectory` - but seems to work without. Adjusted.